src/clojure_tic_tac_toe/rules.clj

```diff
@@ -0,0 +1,33 @@
+(ns clojure-tic-tac-toe.rules
+  (:require [clojure-tic-tac-toe.board :as board]))
+
+(defn- combos [board]
+  [
+   (hash-set (get board 0) (get board 1) (get board 2))
+   (hash-set (get board 3) (get board 4) (get board 5))
+   (hash-set (get board 6) (get board 7) (get board 8))
+   (hash-set (get board 0) (get board 3) (get board 6))
+   (hash-set (get board 1) (get board 4) (get board 7))
+   (hash-set (get board 2) (get board 5) (get board 8))
+   (hash-set (get board 0) (get board 4) (get board 8))
+   (hash-set (get board 2) (get board 4) (get board 6))
+   ])
+
+(defn- win-on-any? [board]
+  (some #(= 1 (count %)) (combos board)))
+
+(defn winner? [board]
+  (if(win-on-any? board)
```

Comment:
dont think you need the if statement here, which means `win-on-any?` is basically `winner?`. does that make sense?